components/actiongroup/stories/actiongroup.test.js

```diff
@@ -25,5 +25,36 @@ export const ActionGroups = Variants({
 			testHeading: "Vertical",
 			vertical: true,
 		},
+		{
+			testHeading: "Overflow",
+			customStyles: { "max-inline-size": "288px" },
+			content: [
+				{
+					iconName: "Edit",
+					iconSet: "workflow",
+					label: "Edit",
+				},
+				{
+					iconName: "Copy",
+					iconSet: "workflow",
+					label: "Copy",
+				},
+				{
+					iconName: "Delete",
+					iconSet: "workflow",
+					label: "Delete",
+				},
+				{
+					iconName: "Cut",
+					iconSet: "workflow",
+					label: "Cut",
+				},
+				{
+					iconName: "Move",
+					iconSet: "workflow",
+					label: "Move",
+				},
+			]
+		}
 	],
```

Comment:
I think we could add some coverage for the icon-only and text-only options. Tabs has those same 3 options (text-only, text + icon, icon-only), so [we used the `stateData`](https://github.com/adobe/spectrum-css/blob/7e8f6114aef049b82904cebf6a112aa5a2be0feb/components/tabs/stories/tabs.test.js#L86) to cover all of them.

<img width="773" alt="Screenshot 2024-10-02 at 1 59 51 PM" src="https://github.com/user-attachments/assets/72a350e8-0b88-4ab8-bb3f-a4bba7b0617a">